src/Twig/MoocElementExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\Twig;
+
+use AppBundle\Entity\Mooc\BaseMoocElement;
+use AppBundle\Entity\Mooc\Quiz;
+use AppBundle\Entity\Mooc\Video;
+use Twig\Extension\AbstractExtension;
+
+class MoocElementExtension extends AbstractExtension
+{
+    public function getTests()
+    {
+        return [
+            new \Twig_SimpleTest('video', function (BaseMoocElement $element) {
```

Comment:
to avoid any future confusion i think it should be `if object is mooc_video`, and not `is video`
on this application another kind of object can be more generic video: `AppBundle\Entity\Media`.

what do you think?